docker-compose.yml

```diff
@@ -1,8 +1,10 @@
 db:
   build: db
+  image: mattermost/mattermost-prod-db
   volumes:
     - ./volumes/db/var/lib/postgresql/data:/var/lib/postgresql/data
-    - /etc/localtime:/etc/localtime:ro
+  environment:
+    - POSTGRES_PASSWORD
```

Comment:
Sorry, I didn't update the README, but yes, you are right.
The README should contain something like:
```
To start the docker-compose you need to feed a POSTGRES_PASSWORD to the environment. You can do it 2 ways:
 - modify the docker_compose with your value of the password
 - POSTGRES_PASSWORD=mypassword docker-compose up
```

I just prefer that than feeding a "default" password, that you'll end up, most of the people will keep.